Latest Project Code - Including Ali's Private Chat

gamecoder95 opened the pull request:
This is the project that we set up on Tashnim's computer. We merged Ali's chat functions manually on her laptop.

tash2cool committed:
latest code for webpage

gamecoder95 committed:
Modified Profiles

gamecoder95 committed:
Merge branch 'tash2' into ahmad2

tash2cool reviewed (approved):
everything looking good so far!

gamecoder95 committed:
Refactored Routes for Testing #66

Separated routes.js into functions for each feature (except for profile
page and some other things, as they couldn't be separated from the
runApp function).
